debian/authd.gdm-authd.pam

```diff
@@ -8,6 +8,7 @@ auth    substack        common-auth
 auth    requisite       pam_nologin.so
 auth    optional        pam_gnome_keyring.so
 
+account [success=ok default=bad ignore=ignore authinfo_unavail=ignore]	pam_authd.so
```

Comment:
I think we are lacking consistency here. Right now, in `debian/pam-configs/authd.in`, we have:
```
Account:
	[default=ignore success=ok user_unknown=ignore]	pam_authd_exec.so AUTHD_DAEMONS_PATH@/authd-pam
```
Despite having a PR that changes it to this new configuration that you are using here, it's not merged yet so I think we should use the same values that we have in `debian/pam-configs/authd.in` for now.